packages/mobile/src/screens/account-screen/AccountDrawer.tsx

```diff
@@ -54,14 +55,16 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
     justifyContent: 'space-between'
   },
   accountImage: {
-    height: spacing(12),
-    width: spacing(12),
-    marginBottom: spacing(3)
+    height: spacing(12) + 2,
```

Comment:
should we accept partial sizes instead of this?

Reply:
Yeah will do, it's cause border width, I wonder if there is a clear way to express that